src/braket/circuits/result_type.py

```diff
@@ -209,32 +209,32 @@ def __init__(
         super().__init__(ascii_symbols)
         self._observable = observable
         self._target = QubitSet(target)
-        if not self._target:
-            if self._observable.qubit_count != 1:
-                raise ValueError(
-                    f"Observable {self._observable} must only operate on 1 qubit for target=None"
-                )
-        elif isinstance(observable, Sum):  # nested target
-            if len(target) != len(observable.summands):
-                raise ValueError(
-                    "Sum observable's target shape must be a nested list where each term's "
-                    "target length is equal to the observable term's qubits count."
-                )
-            self._target = [QubitSet(term_target) for term_target in target]
-            for term_target, obs in zip(self._target, observable.summands):
-                if obs.qubit_count != len(term_target):
+        if self._target:
```

Comment:
What if `target` is an empty list? This will branch to the `target=None` check below at line 237. Should we add a `target=[]` test case?

Reply:
Existing behavior is to treat `[]` the same as `None`, and this preserves this behavior. I'm also fine with casting a wider net too though:
```suggestion
        if target is not None:
```

Reply:
Looks like that was intentional in the past; I'll revert